spec/factories/draft_taxon.rb

```diff
@@ -0,0 +1,12 @@
+FactoryBot.define do
```

Comment:
This is very similar to the Taxon factory - could this not better be a trait of the taxon factory instead?